internal/flags/flags.go

```diff
@@ -39,4 +41,5 @@ const (
 	DisableColorsUsage = "disable output styling"
 	OutputFormatUsage  = `set the output format, available options: [json]`
 	OutputTargetUsage  = "write output to the specified filepath"
+	TelemetryModeUsage = `set the telemetry settings. This setting is sticky. Available options: ["on", "off", "stdout"]`
```

Comment:
[nit] `enable or disable telemetry (this setting is remembered), available options: ["off", "on"]`